samples/audio-video-player/mp4_pull_demuxer.js

```diff
@@ -1,3 +1,5 @@
+import {PullDemuxerBase, AUDIO_STREAM_TYPE, VIDEO_STREAM_TYPE} from '../library/pull_demuxer_base.js'
```

Comment:
Does this not run in a worker?

Reply:
It does run in a worker (samples/audio-video-player/media_worker.js). Am I missing something?

Reply:
Ah, I guess you were referring to this comment
https://github.com/w3c/webcodecs/pull/549#discussion_r968914714

I'll answer there.